contrib/fwupd.spec.in

```diff
@@ -484,6 +510,52 @@ done
 %config(noreplace)%{_sysconfdir}/fwupd/remotes.d/fwupd-tests.conf
 %endif
 
+%files qubes-vm
+%FWUPD_QUBES_DIR/fwupd-download-updates.sh
+%FWUPD_QUBES_DIR/fwupd_usbvm_validate.py
+
+%files qubes-dom0
+%FWUPD_QUBES_DIR/src/fwupd_receive_updates.py
+%FWUPD_QUBES_DIR/src/qubes_fwupdmgr.py
+%FWUPD_QUBES_DIR/src/fwupd-dom0-update
+%FWUPD_QUBES_DIR/src/qubes_fwupd_heads.py
+%FWUPD_QUBES_DIR/src/__init__.py
+%FWUPD_QUBES_DIR/test/fwupd_logs.py
+%FWUPD_QUBES_DIR/test/test_qubes_fwupdmgr.py
+%FWUPD_QUBES_DIR/test/test_qubes_fwupd_heads.py
+%FWUPD_QUBES_DIR/test/__init__.py
+%FWUPD_QUBES_DIR/test/logs/get_devices.log
+%FWUPD_QUBES_DIR/test/logs/get_updates.log
+%FWUPD_QUBES_DIR/test/logs/help.log
+%FWUPD_QUBES_DIR/test/logs/firmware.metainfo.xml
+%FWUPD_QUBES_DIR/test/logs/metainfo_name/firmware.metainfo.xml
+%FWUPD_QUBES_DIR/test/logs/metainfo_version/firmware.metainfo.xml
+
+%post qubes-dom0
+# Replace the python shebangs if fedora is older then 25
+%if 0%{?fedora} <= 25
+sed -i 's/python3/python36/g' %FWUPD_QUBES_DIR/src/qubes_fwupdmgr.py
+sed -i 's/python3/python36/g' %FWUPD_QUBES_DIR/src/fwupd_receive_updates.py
+sed -i 's/python3/python36/g' %FWUPD_QUBES_DIR/src/qubes_fwupd_heads.py
+sed -i 's/python3/python36/g' %FWUPD_QUBES_DIR/test/test_qubes_fwupdmgr.py
+sed -i 's/python3/python36/g' %FWUPD_QUBES_DIR/test/test_qubes_fwupd_heads.py
+sed -i 's/python3/python36/g' %FWUPD_QUBES_DIR/test/fwupd_logs.py
+%endif
```

Comment:
same; Fedora 25 is EOL and dead dead dead, no?

Reply:
As above.

Reply:
Deleted. I've left support only for Qubes R4.1